go/tools/bzltestutil/coverage_visitor.go

```diff
@@ -0,0 +1,179 @@
+package bzltestutil
+
+// This is a stripped-down version of https://github.com/golang/go/blob/633dd1d475e7346b43d87abc987a8c7f256e827d/src/cmd/covdata/dump.go
+// which supports only the `textfmt` sub-command.
```

Comment:
Could you add a TODO here to merge this with the lcov conversion routine? If we already copy this (which I think is very reasonable), we might as well emit lcov directly instead of going through the textual format.